backend/inspirehep/records/models.py

```diff
@@ -132,3 +135,48 @@ class InstitutionLiterature(db.Model):
     institution_paper = db.relationship(
         RecordMetadata, backref="institutions", foreign_keys=[literature_uuid]
     )
+
+
+class AuthorSchemaType(enum.Enum):
+    INSPIRE_ID = "INSPIRE ID"
+    INSPIRE_BAI = "INSPIRE BAI"
+    ORCID = "ORCID"
+    JACOW = "JACOW"
+    KAKEN = "KAKEN"
+    ARXIV = "ARXIV"
+    CERN = "CERN"
+    DESY = "DESY"
+    GOOGLESCHOLAR = "GOOGLESCHOLAR"
+    VIAF = "VIAF"
+    RESEARCHERID = "RESEARCHERID"
+    SCOPUS = "SCOPUS"
+    SPIRES = "SPIRES"
+    WIKIPEDIA = "WIKIPEDIA"
+    SLAC = "SLAC"
+    TWITTER = "TWITTER"
+    LINKEDIN = "LINKEDIN"
+    collaboration = "collaboration"
+
+
+class RecordsAuthors(db.Model):
+    __tablename__ = "records_authors"
+    __table_args__ = (
+        db.Index("ix_authors_records_record_id", "record_id"),
+        db.Index(
+            "ix_authors_records_author_id_id_type_record_id",
+            "author_id",
+            "id_type",
+            "record_id",
+        ),
+    )
+    id = db.Column(db.Integer, primary_key=True)
+    author_id = db.Column(Text, nullable=False)
+    id_type = db.Column(
+        ENUM(*[key.value for key in AuthorSchemaType], name="enum_author_schema_type"),
```

Comment:
is this the same type as the one in the alembic migration?

Reply:
Argh, right. It's not. I up[dated it now. With this enum I had a lot of problems as normal way of defining it wasn't working properly. Sqlalchemy is setting enum names not based on values but based on enum keys :/